### DISCUSSION TITLE
Refund request

### DISCUSSION THREAD

**Discussion by skyko03** [aclap-dev/video-downloadhelper#1172] [Help and Q&A]
### Your question

I can't download swag videos, it's stopping me from working

### Addon & CoApp version and details

email address
skykokiu@hotmail.com

60578316bf

thanks

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by skyko03** [comment]
Thank you very much

取得 Android 版 Outlook<https://aka.ms/AAb9ysg>
________________________________

**Comment by skyko03** [comment]
Sorry one more questions i saw my bank statement pay two time Usd18 will also refund or no need to care about it thanks

取得 Android 版 Outlook<https://aka.ms/AAb9ysg>
________________________________
